### DISCUSSION TITLE
Several Faction not loaded in H&M

### DISCUSSION THREAD

**Discussion by Pochtron** [Vdauphin/HeartsAndMinds#1201] [Question and answer]
Hi Vdauphin, I hope you are going well!
I'm currently having trouble with ennemy factions, for instance most of the RHS Faction like 230, 232, 242, 229, or even project OPFOR faction like 178 aren't loading and I just have by default an Independant Faction I don't understand why because 213 for Russia VDV is working well and I have all RHS mods activated. Other question, I dont find US Army - W for woodland in the list is it unaccessible for now?

Thanks in advance for your answers!

Pochtron

**Comment by Vdauphin** [comment]
Hello,

![20211016114858_1](https://user-images.githubusercontent.com/14364400/137583093-b65b2446-7318-4240-8e63-7c1b573fd77d.jpg)
![20211016114840_1](https://user-images.githubusercontent.com/14364400/137583094-fc19ff49-9062-4e53-8fd2-215797321560.jpg)

Everything is fine on last released

Cheers

**Reply by Pochtron** [reply]
Hi, Thanks for you answer, I've seen that I didn't had the last version of H&M indeed, so I update it properly and I tried a test on original file (altis) by just changing the faction for RHS USAF US Army - W = 242 (not sure about the number), and the faction doesn't want to load either. My solution was to use RHS USAF USMC - W but I'm still interested to know if it's a problem on your side too or not, in this case, I will wait a fix patiently no worries :)

**Reply by Vdauphin** [reply]
> so I update it properly and I tried a test on original file (altis) by just changing the faction for RHS USAF US Army - W = 242 

this faction is `west` side, you can't fight a faction with the same side as you 
You need to follow this http://vdauphin.github.io/HeartsAndMinds/faction_player

**Comment by Pochtron** [comment]
Oh well I didn't seen that already thanks a lot for your answer, it make
sense! Problem solved
